.github/workflows/linux.yml

```diff
@@ -30,16 +30,12 @@ jobs:
 
           # Raspberry Pi OS is close enough to Debian to test just one of them.
           # Its architecture is different, though, and covered by the Docker job.
-          - debian:11
```

Comment:
Are you two kidding me?

"should something break on Debian 11", it's **the GHAs' only purpose** to tell that at PR time, not on release day morning when someone looks at package builds started yesterday! So "extended support but shouldn't be relevant for the GHAs" is complete nonsense.

While on Debian 11, no – it's not "old enough that we could stop building for it"! What does that term even mean? RHEL 8 is older and still supported. Whatever. Debian 11 is still in our support matrix on the website – no surprise given we support "[Full Support and LTS](https://github.com/Icinga/icinga-packaging/blob/master/doc/04-OS-EOL.md)" which, in this case, ends on [2026-08-31](https://wiki.debian.org/DebianReleases#Production_Releases).